package.json

```diff
@@ -41,7 +41,7 @@
     "test:functional-nightly-preview": "yarn playwright install chromium && yarn playwright test --project=preview",
     "test:accessibility-nightly": "yarn playwright install chromium && yarn playwright install firefox && yarn playwright install webkit && yarn playwright test --grep @accessibility",
     "test:smoke": "yarn npm audit --recursive --environment production && MOCHAWESOME_REPORTFILENAME=smoke REPORT_DIR=test-results/smokeTest REPORT_FILE=test-results/smokeTest/results.xml codeceptjs run --grep '@smoke-tests' --reporter mocha-multi --verbose",
-    "test:local": "yarn playwright test --grep @local",
+    "test:local": "yarn playwright test --headed --debug --project=preview --retries=1 --grep @local",
```

Comment:
retries property is set in the project preview in playwright config file. Do we want to override it here?

Reply:
yes, this script for the local set up  and does not want 3 retries. It is not called from any where but used only for the local .